pkg/handlers/webhookrhobsreceiver_test.go

```diff
@@ -347,6 +333,7 @@ var _ = Describe("RHOBS Webhook Handlers", func() {
 							},
 						},
 					}
+					testMFN.Spec.FleetNotification.ResendWait = 24
```

Comment:
We're now going off the fleetnotification's resendwait, not the one specific for a record.